proto/topodata.proto

```diff
@@ -333,6 +333,27 @@ message ShardReplication {
   repeated Node nodes = 1;
 }
 
+// ShardReplicationProblem describes the problem being fixed by
+// ShardReplicationFix.
+message ShardReplicationProblem {
```

Comment:
nit with the name - It seems more intuitive to me to call this `ShardReplicationError`.  Problem is correct as well, but in my view it is an uncommon word in this context. We normally talk about error codes and errors. 

Was there a particular reason to name it `Problem`?

Reply:
Good question! I don't have a particularly strong preference either way. The reason I went with "Problem" is that it's not _currently_ an error; it _used_ to be an error, but we just fixed it.

Separately (embarrassingly), I was going to say that it was because the `-help` for `vtctl ShardReplicationFix` read "and fixes the first _problem_ [ed. not 'error'] it encounters." I could have _sworn_ it did, but in fact it says "and fixes the first _error_ it encounters," so maybe I should go with "Error".

But, in either case, that was my thought process. Let me know what you think!

Reply:
I see. I think I have taste preference for error and consistent with "and fixes the first error it encounters". I'm fine either way. You can take the final call on this